apps/common-lib/src/main/scala/com/gu/typerighter/controllers/PandaAuthController.scala

```diff
@@ -24,6 +24,8 @@ abstract class PandaAuthController(
     with HMACAuthActions
     with BaseController
     with Loggable {
+  override val apiGracePeriod: Long = 2 * 60 * 60 * 10000 // two hours worth of milliseconds
```

Comment:
I think you should be multiplying by one thousand rather than ten thousand here

Reply:
Yikes well spotted - sorry I missed this!

Reply:
Thanks for spotting!
```suggestion
  override val apiGracePeriod: Long = 2 * 60 * 60 * 1000 // two hours worth of milliseconds
```